config/example.config.toml

```diff
@@ -29,6 +29,8 @@ color = true
 # Maximum number of firmware exits before terminating.
 # No maximum cap if not present
 max_firmware_exits = 400
+# Option to compile using dev profile (release profile is set by default)
+debug_compile = false
```

Comment:
Usually this is simply called `debug` in other projects, so I would tend to use the same name.

And sorry for changing my mind, but after reflection I think we should compile in debug by default, not release (see comment on the issue).